tests/checks/cd.fish

```diff
@@ -194,6 +194,12 @@ cd $old_path
 cd file
 cd $old_path
 
+# Test that going up to the root directory using .. works
+cd /(string split --no-empty -f 1 / (pwd))
```

Comment:
Not too happy with this, but couldn't come up with a better way to make this work with arbitrary directories in `/`.

Reply:
How about something boring like `for i in (seq 100); cd ..; end` ?

Reply:
Hmm, I can imagine failure cases where going to `..` in the root directory would normalize it again.